pinot-core/src/test/java/com/linkedin/pinot/core/segment/index/ColumnMetadataTest.java

```diff
@@ -67,6 +67,17 @@ public SegmentGeneratorConfig CreateSegmentConfigWithoutCreator()
     return config;
   }
 
+  public SegmentGeneratorConfig createSimpleSegmentConfig()
```

Comment:
Can you move this method to DefaultSegmentNameGeneratorTest? the avro file should be accessable from there as well.